website/content/docs/connect/gateways/mesh-gateway/wan-federation-via-mesh-gateways.mdx

```diff
@@ -42,6 +40,13 @@ gateways](/docs/connect/gateways/mesh-gateway/service-to-service-traffic-datacen
 
 [![WAN federation with mesh gateways](/img/wan-federation-connectivity-mesh-gateways.png)](/img/wan-federation-connectivity-mesh-gateways.png)
 
+## Requirements
+
+- Consul 1.8.0+
+- Mesh federation between Consul datacenters: all datacenters must be connected to all other datacenters
+   - Other topologies, such as [hub-and-spoke enabled by Consul Enterprise network areas](https://learn.hashicorp.com/tutorials/consul/federation-network-areas),
+     cannot be used
```

Comment:
```suggestion

```

Reply:
I'm confused, why remove `cannot be used`?

Reply:
My previous suggestion would have removed it: https://github.com/hashicorp/consul/pull/12290#discussion_r801932914